remoulade/api/main.py

```diff
@@ -32,6 +33,18 @@ def get_state(message_id):
     return data.asdict()
 
 
+@app.route("/messages/cancel/<message_id>", methods=["GET", "POST"])
```

Comment:
I don't really like GET requests changing the state of the app, just accept POST.